app/controllers/users/bank_accounts_controller.rb

```diff
@@ -0,0 +1,47 @@
+class Users::BankAccountsController < ApplicationController
+  after_filter :verify_authorized, except: [ :index ]
+  before_action :has_list_permission?, only: [ :index ]
+
+  def index
+    head :ok
+  end
+
+  def new
+    @bank_account = BankAccount.new(user: current_user)
+    authorize @bank_account
+
+    render nothing: true
```

Comment:
At the index action you're using `head :ok` and here you're using `render nothing: true`. Why the difference?. Both do the same and `head...` is preferred over `render nothing...`.